libmscore/beam.cpp

```diff
@@ -388,9 +388,12 @@ void Beam::layout1()
                         _up = false;
                   else if (c1) {
                         Measure* m = c1->measure();
+                        Fraction totalTicks = Fraction(0, 1);
+                        for (size_t i = 0; i < _elements.size(); ++i)
+                              totalTicks += _elements[i]->globalTicks();
```

Comment:
Now the question is whether this possibly costly operation should better get moved into the branch were we detect that we have voices and so save the extraa computation on the other 2-3 cases?

Reply:
I'm not sure what you mean by that... before I saw this I created a function Beam::totalTicks() const, is this what you want?

Reply:
Yes, even vetter then my initial idea. Could even be static